hazelcast/src/main/java/com/hazelcast/map/eviction/MapEvictionPolicy.java

```diff
@@ -48,8 +48,8 @@
      * {@inheritDoc}
      *
      * <p>
-     *     If you prefer to evict the 1st entry then return positive number, if the 2nd entry is a better candidate
-     * then return a negative number or return 0 if both entries are equally good candidates for eviction
+     * If you prefer to evict the 1st entry then return positive number, if the 2nd entry is a better candidate
```

Comment:
`If you prefer to evict the 1st entry then return positive number` should be `If you prefer to evict the 1st entry then return a negative number`